### PR TITLE
fix(DB/Creature): Centipaar Tunneler attack on summon

### PR DESCRIPTION
**Pull request by febaut**
Centipaar Tunnelers from Water Survey quest now attack the player after collecting samples. 

The issue suggests that maybe is a missing InhabitType but is not, tried adding water and still the same.

## Changes Proposed:
-  Adding a new SMART SCRIPT to Centipaar Tunneler.  Entry: 5459

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9984

## Tests Performed:

- 
- 

## How to Test the Changes:

1. Add the smart script row
2. .go xyz -7218 -2913 6.5 1 1
3. .additem 8584
4. Use the item. The mobs should insta attack you.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1641245804524923284.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641245804524923284');
+
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES (5459, 0, 2, 0, 54, 0, 100, 0, 0, 0, 0, 0, 0, 46, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Centipaar Tunneler - On Just Summoned - Move Forward 1 Yards');
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1641245804524923284.sql:3]: INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
(5459, 0, 2, 0, 54, 0, 100, 0, 0, 0, 0, 0, 0, 46, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Centipaar Tunneler - On Just Summoned - Move Forward 1 Yards');

Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1641245804524923284.sql:3]: Add line Break for readability. One extremely long line does make things a bit problematic on the human eye and other reviewers. We prefer to keep it all on one page.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Comment by Nyeriah** [comment]
>> Centipaar Tunnelers from Water Survey quest now attack the player after collecting samples.
>> I preferred "Move one Yard" rather "Attack Start" cause it causes a double auto attack each mob meaning a lot of damage.

This assumption is not correct, attack start won't cause them to hit you twice, it just tells the creature to start attacking you. In this specific case, you could use the action to start attacking the summoner.

File: data/sql/updates/pending_db_world/rev_1641245804524923284.sql (hunk 1 above)
Comment (febaut) [reply data/sql/updates/pending_db_world/rev_1641245804524923284.sql:3]: Thanks a lot. Will take it into account next time.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1641245804524923284.sql:3]: Not next time. This time.

Also you always need to a DELETE statement before an INSERT
You can learn more about SQL stuff here https://www.azerothcore.org/wiki/sql-standards :)

**Comment by febaut** [comment]
Is not an "assumption". Please go and try with  attack start. You'll get 4 hits at once instead of 2.

**Comment by Kitzunu** [comment]
This is not an acceptable fix. This is a hack.

If there now is an issue with Attack Start then we should fix that. And not hack fix through things in the database.

**Comment by febaut** [comment]
Updated with Start Attacking. Thanks for the recommendations. A bit rude to someone that is just trying to help, but if it's usless, okay ^^.

**Comment by Kitzunu** [comment]
I apologize if I seemed rude. I appreciate the effort. But this is not the right way of fixing the issue

**Comment by acidmanifesto** [comment]
Added Need Sniff label for further clarification in how it should be handled.

**Comment by Nyeriah** [comment]
If the creature attacks the summoner this is fine. I don’t understand what needs to be sniffed about this. The only part incorrect about this PR was forcing the creature move on top of the player to trigger the attack reaction instead of doing it through the summon event.

**Comment by temperrr** [comment]
@febaut 

Thanks you so much for your contribution! I have tested it and it is working as described!

**Comment by acidmanifesto** [comment]
> @febaut
> 
> Thanks you so much for your contribution! I have tested it and it is working as described!

The Sql still needs to be adjust and we need a delete query as well before the insert.

**Comment by febaut** [comment]
> > @febaut
> > Thanks you so much for your contribution! I have tested it and it is working as described!
> 
> The Sql still needs to be adjust and we need a delete query as well before the insert.

I did the changes already.

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
> > > @febaut
> > > Thanks you so much for your contribution! I have tested it and it is working as described!
> > 
> > 
> > The Sql still needs to be adjust and we need a delete query as well before the insert.
> 
> I did the changes already.

I do apologize. I really did not see the delete line at all when i made the comment. hitting refresh a couple of times after another staffer informed me it was there, it then showed. 
Thank you for the PR.

**Comment by Nyeriah** [comment]
Thanks for your PR! 🚀

**Pull request merged**